Simplified/AxisService/NYPLAxisPackagePathPrefixProvider.swift

```diff
@@ -37,6 +37,10 @@ class NYPLAxisPackagePathPrefixProvider: NYPLAxisPackagePathPrefixProviding {
       .deletingLastPathComponent()
       .absoluteString
     
+    if packagePathPrefix == "./" {
```

Comment:
is it a concern that even if we return nil here, we have modified the state of this class with something that looks wrong?

Also, can you add a comment to explain the rationale here? E.g. why is this bad/incorrect? it seems very specific.

Reply:
If we return nil, it means the package and it's content are supposed to be in the root directory of the book instead of a subdirectory.
Added comment explaining the reasoning.